src/components/Sidebar.js

```diff
@@ -8,35 +8,22 @@ const StyledSidebar = styled.div`
   justify-content: center;
   align-items: center;
 `
+const getAllFileNames = () => {
```

Comment:
I think there's a better name we can use than getFileNames for this function. Also this function should probably take care of the res => res.json so that the component does not need to. 

Let's say multiple components used this function, what would be your approach?

Reply:
Good call. Will implement. If multiple components used this function then I would move it out into a helper function file so those components would have access to it as well and to keep things DRY.